source/configure/configuration-settings.rst

```diff
@@ -6052,6 +6052,19 @@ This setting has been added as a requirement to support `Collapsed Reply Threads
 
    Enabling this configuration setting doesn’t retroactively follow threads for older actions taken prior to the setting being enabled. For example, if a user comments on an old thread, they automatically follow the thread if this setting is enabled regardless of whether the thread existed when this setting was disabled.
 
+Developer Flags
+^^^^^^^^^^^^^^^
+
+|all-plans| |self-hosted|
+
+This setting isn't available in the System Console and can only be set in ``config.json``.
+
+This configuration setting specifies a list of strings where each string is a flag used to set the content security policy for the Mattermost web app. Each flag must be in the format ``flag=true`` (e.g. ``unsafe-eval=true,unsafe-inline=true``). This configuration setting is disabled by default and requires `Developer Mode <https://docs.mattermost.com/configure/configuration-settings.html#enable-developer-mode>`__ to be enabled.
```

Comment:
For now, only `unsafe-eval` and `unsafe-inline` are supported. Can we make that explicit? It might be worth documenting as:
```
unsafe-eval: Adds the `unsafe-eval` CSP directive to the root webapp, allowing increased debugging in developer environments.
unsafe-inline: Adds the `unsafe-inline` CSP directive to the root webapp, allowing increased debugging in developer environments.
```

We should probably emphasize that these values shouldn't typically be enabled in production.

Reply:
Great clarification. Thanks!